chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -143,19 +143,19 @@ endif::VK_VERSION_1_1,VK_KHR_device_group,VK_KHR_swapchain_mutable_format[]
 | pname:initialLayout           | ename:VK_IMAGE_LAYOUT_UNDEFINED
 |====
 
-The pname:surface must: not be destroyed until after the swapchain is
+The pname:pCreateInfo->surface must: not be destroyed until after the swapchain is
```

Comment:
@oddhack - Is this proper markup? "pname:pCreateInfo->surface"  ?

Reply:
Yes, that's correct.

Reply:
Use this   slink:(struct name)::pname:surface

Reply:
Since pCreateInfo is a named parameter here, best to stay with pCreateInfo->surface. The VkStructName::member syntax is mostly for when you don't *know* the name of the structure in question, for example when referring to a structure up the pNext chain, or the CreateInfo structure used in the past to create the object you're now operating with.